lib/workers/repository/dependency-dashboard.ts

```diff
@@ -316,6 +317,9 @@ export async function ensureDependencyDashboard(
       'This repository currently has no open or pending branches.\n\n';
   }
 
+  issueBody += PackageFiles.getDashboardMarkdown(config);
+  PackageFiles.clear();
```

Comment:
Move this logic into the main repository worker logic. In future it may be more than just the dashboard which uses the package files data.

Reply:
Moved logic into `finaliseRepo`